addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3014,7 +3016,8 @@ var SnippetsMenu = Widget.extend({
             !$(range.commonAncestorContainer).parents('#wrapwrap, .iframe-editor-wrapper .o_editable').length ||
             $(range.commonAncestorContainer).parent('[data-oe-model]:not([data-oe-type="html"]):not([data-oe-field="arch"])').length ||
             (e && $(e.target).closest('.fa, img').length ||
-            this.options.wysiwyg.lastMediaClicked && $(this.options.wysiwyg.lastMediaClicked).is('.fa, img'))
+            this.options.wysiwyg.lastMediaClicked && $(this.options.wysiwyg.lastMediaClicked).is('.fa, img')) ||
+            (selection.focusNode.parentElement.tagName !== 'A' && this.activatedSnippet && !this.activatedSnippet.isContentEditable)
```

Comment:
Not sure to understand this commit, I also tried it and I cannot put some word of a field description in bold anymore :grimacing:

Reply:
At first I wanted to hide the toolbar for inputs, which is what is required from the spec, but I couldn't get the element that was focused from the selection, only its parent... So I went for hiding the toolbar when an element that is not a link and not editable is selected. It suits the spec with hiding the toolbar on inputs but it doesn't seem to be the good approach either, which description field is now hiding the toolbar ? I tested on the product description but the text style was not editable before either

Reply:
The form builder's form fields. You can add a description to them.

Reply:
Ok, thanks, I will find another way to remove the toolbar for inputs